Add information about not to perform the install step (#204)
mattermost/mattermost-plugin-zoom: docs/installation/zoom-configuration/zoom-setup-oauth.md

You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and use it to create meetings and pull user data.

**Account-level Apps** require **an admin to authorize access to all users accounts within the Zoom account**. Individual users in Mattermost are verified by checking their Mattermost email and requesting their Personal Meeting ID via the Zoom API. The user's emails in Zoom and Mattermost accounts should match up. If you prefer for each end user to authorize individually, you should create a Zoom [User-Level App](zoom-setup-user-level-app.md).

## Create an app for Mattermost

1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.

> 2. In the top left click on **Develop** and then **Build App**.

Reviewer: ```suggestion
2. In the top left select **Develop** and then **Build App**.
```

3. Select **OAuth** in **Choose your app type** section.
4. Enter a name for your app.
5. Choose **Account-level app** as the app type.

> 6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.

Reviewer: ```suggestion
6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin supports apps that are published in the Zoom Marketplace. 
```

## Configure your new OAuth app to work with Mattermost

If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.

1. Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
  * These will be needed during [Mattermost Setup]().
2. Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Whitelist URL**.
  * `SITEURL` should be your Mattermost server URL.

![App Credentials screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-34-13%20%282%29.png)

## Add user scopes to the app

Select **Scopes** and add the following scopes: **meeting:read**, **user:read**.

![Scopes screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-37-47%20%282%29.png)

## Do not perform the install step

Zoom has one last option called **Install**. There is no need to perform this action. However, if you perform this action inadvertently, you'll see an error on Mattermost. This is expected.

## Deauthorization

> This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you will be able to set up a Deauthorization URL.

Reviewer: ```suggestion
This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you can set up a deauthorization URL.
```
